### DISCUSSION TITLE
Dates for Spring/Summer Collection 2022

### DISCUSSION THREAD

**Discussion by clearcoding** [aws-lug/aws-lug-meetups#4] [Ideas]
Add some ideas for next dates for upcoming session.
We can then stcik them in a poll and try and get some community consensus

**Comment by pmgledhill102** [comment]
We tend to stick to Thursdays...

- Thu 26th May
- Thu 2nd June // Bank Holiday //
- Thu 9th June
- Thu 16th June
- Thu 23rd June
- Thu 30th June
- Thu 7th July
- Thu 14th July
- Thu 21st July
- Thu 28th July
- Thu 4th August
- Thu 11th August
- Thu 18th August
- Thu 25th August

**Reply by clearcoding** [reply]
as a cadence I think once every 2 months is reasnable

**Reply by clearcoding** [reply]
If we can't get a decent speaker or content then a social is a good fall back

**Comment by pmgledhill102** [comment]
Tuesday 31st May
Thursday 21st July
Tuesday 13th Sept
Thursday 3rd November
